documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -260,7 +323,23 @@ For the same _ReferenceElement,_ the payload is minimized to the following in ca
 }
 ----
 
-For a _File_ named “Document”, the payload is minimized to the following:
+If the _Reference_ itself is the subject for ValueOnly the serialization is minimized to the following:
```

Comment:
This comment is actually on unchanged line 313, the expected result for serializing a ReferenceElement.

This serialization is wrong as it still contains the idShort of the element, while it should only contain the value, i.e. I would expect

````JSON
{
    "type": "ExternalReference",
    "keys": [
        {
            "type": "GlobalReference",
            "value": "0173-1#02-BAA120#008"
        }
    ]
}
```